Formula/golangci-lint.rb

```diff
@@ -0,0 +1,62 @@
+class GolangciLint < Formula
+  desc "Fast linters runner for Go"
+  homepage "https://golangci.com"
+  url "https://github.com/golangci/golangci-lint.git",
+    :tag      => "v1.29.0",
+    :revision => "6a689074bf17fd4ae1db779a74dba821a162b6c8"
+  license "GPL-3.0"
+
+  depends_on "go" => :build
```

Comment:
The package needs go as runtime too.

Reply:
Can you give me a bit more context about this? I never meet any "runtime" concept while working with brew formulas, more of that go itself doesn't require runtime, as it compiles to binary. Can you please give me more info about that? Maybe some existing formulates that use this feature?

Reply:
Well, the text is failing because it doesn't have go installed. So when people run this tool they'll need go installed.

Therefore just setting the dependency to `:build` is incorrect.

Reply:
Thank you, it's clear now.